base/migrations/0024_auto_20230507_2205.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.12 on 2023-05-07 19:05
+
+from django.db import migrations, models
+
+def set_avialable_state(apps, schema_editor):
+    model = apps.get_model('base','grouptrainingday')
+    
+    for mm in model.objects.all():
+       if not mm.is_available:
+            mm.available_status = 'N'
+         
+       mm.save()
```

Comment:
```suggestion
    model.objects.filter(is_available=False).update(available_status='N')
```
Сделает один запрос к БД вместо `N` запросов